src/docking/search_nci.f90

```diff
@@ -1065,11 +1066,19 @@ subroutine restart_gff(env, mol, calc)
       call gfnff_param_dealloc(calc%topo)
       call newD3Model(calc%topo%dispm, mol%n, mol%at)
       call gfnff_set_param(mol%n, calc%gen, calc%param)
+      if (allocated(calc%neigh%nb)) deallocate(calc%neigh%nb)
       if (.not. allocated(calc%neigh%nb)) allocate (calc%neigh%nb(calc%neigh%numnb, mol%n, calc%neigh%numctr), source=0)
```

Comment:
After the enforced deallocation in line 1069, `if (.not. allocated(calc%neigh%nb))` is not necessary again.

Reply:
It is removed now.